### ISSUE TITLE
LOAD and SAVE should respect current BANK

### ISSUE THREAD

**Issue by mist64** [commanderx16/x16-rom#345]

**Comment by irmen** [comment]
There's a catch, I think.
It's pretty hard already to grok all LOAD, VLOAD , BLOAD differences and arguments
What will happen if we're going to change those once again.... ?

**Comment by ZeroByteOrg** [comment]
I presume this refers to BASIC leaving the active bank unchanged after completing the LOAD, and not the Kernal routines? I say this because regarding the Kernal LOAD function, it's necessary to know what the ending bank was after a LOAD so I'd hope that the Kernal would continue to leave the new bank active.

**Comment by irmen** [comment]
Yeah this should refer to the BASIC commands LOAD and SAVE.   Because there is now a BANK statement but that only works for SYS POKE and PEEK
